### ISSUE TITLE
The Driver needs to be reinstalled --> Installation failed because of an I/O Error

### ISSUE THREAD

**Issue by michox** [ShadySquirrel/ViPER4Android#38]
Hey is there a way to get it work on a custom os? 
newest Release 11 and Magisk 16.0
Oneplus 3T on Resurrection Remix (Android 8.1)
no other modules installed
no other sound mod installed
Installation Log:
[install_log_20180412_073820[1].log](https://github.com/ShadySquirrel/ViPER4Android/files/1901496/install_log_20180412_073820.1.log)
Magisk Log:
[magisk_log_20180412_074450[1].log](https://github.com/ShadySquirrel/ViPER4Android/files/1901521/magisk_log_20180412_074450.1.log)
Audio files:
/vendor/etc/audio_effects.xml:
[audio_effects.xml[1].txt](https://github.com/ShadySquirrel/ViPER4Android/files/1901524/audio_effects.xml.1.txt)
/system/etc/audio_effects.conf
[audio_effects.conf[1].txt](https://github.com/ShadySquirrel/ViPER4Android/files/1901527/audio_effects.conf.1.txt)
/system/vendor/etc/audio_effects.xml: (should be the same as the first but who knows...)
[audio_effects.xml[1].txt](https://github.com/ShadySquirrel/ViPER4Android/files/1901533/audio_effects.xml.1.txt)

basically the installation runs without errors, but once i start v4a after restart, it says the driver needs to be reinstalled, and then immediately gives me an I/O Error.
I tried with permissive and enforcing SEPolicy without change

If you need any more Info, please let me know
